cypress/e2e/status.cy.js

```diff
@@ -23,15 +23,88 @@ describe('home page loads', () => {
 
   })
 
-  it('validates required feilds', () => {
+  it('Status page has no detectable a11y violations on load', () => {
+    cy.checkA11y()
+  })
+})
+
+describe('ESRF field validation', ()=>{
+  beforeEach(() => {
+    cy.visit('/status')
+  })
+
+  it('validates valid ESRF',()=>{
+    cy.get('#esrf').type('35934S87')
+    cy.get('#form-get-status > button').click()
+    cy.get('#input-esrf > span').should('not.exist')
+  })
+
+  it('validates empty ESRF error',()=>{
     cy.get('#form-get-status > button').click()
     cy.get('#input-esrf > span').should('contain.text', 'required')
```

Comment:
testing for specific text that could change (while you didn't add this, good to fix my earlier mistake at the same time)